app/models/api/EligibilitySubmissionData.scala

```diff
@@ -64,21 +64,20 @@ object EligibilitySubmissionData {
           }
         ) and
         json.validate[TurnoverEstimates](TurnoverEstimates.eligibilityDataJsonReads) and
-        json.validate[CustomerStatus](CustomerStatus.eligibilityDataJsonReads) and
         (json \ "businessEntity").validate[PartyType] and
         (json \ "registrationReason").validateOpt[String] and
         (json \ "registeringBusiness").validate[String] and
         json.validateOpt[TogcCole](TogcCole.eligibilityDataJsonReads).orElse(JsSuccess(None))
-      ) ((threshold, exceptionOrException, turnoverEstimates, customerStatus, businessEntity, registrationReason, registeringBusiness, optTogcCole) =>
+      ) ((threshold, exceptionOrException, turnoverEstimates, businessEntity, registrationReason, registeringBusiness, optTogcCole) =>
       EligibilitySubmissionData(
         threshold,
         exceptionOrException,
         turnoverEstimates,
-        customerStatus,
+        MTDfB,
```

Comment:
I'd remove the "customerStatus" parameter from the EligibilitySubmissionData model entirely, rather than defaulting its value. MTDfB is mandatory from the 1st of April for everybody so we won't ever need it here again. Then you can hard-code it in SubmissionAuditBuilder and AdminBlockBuilder.

You could also then probably remove the CustomerStatus model and just have a constant containing the value "2" somewhere as we'll just be sending that in all cases